operations/_scripts/deploy/deploy.sh

```diff
@@ -30,6 +30,12 @@ if [[ "$CREATE_DOMAIN" == "true" ]]; then
   /bin/bash $GITHUB_ACTION_PATH/operations/_scripts/generate/generate_acm_tf.sh
 fi
 
+# Generate Ansible
+# -------------------------------- #
+/bin/bash $GITHUB_ACTION_PATH/operations/_scripts/generate/ansible/generate_st2_conf.sh
```

Comment:
The new file is just a few lines.
Could we include that `cp` logic here instead of creating a new bash script?

Also the `Generate Ansible` or `generate_st2_conf` is not relevant as we're just sharing the Ansible vars from the user's side.